include/cantera/thermo/GibbsExcessVPSSTP.h

```diff
@@ -94,28 +94,7 @@ class GibbsExcessVPSSTP : public VPStandardStateTP
     //! @{
 
 protected:
-    /**
-     * Calculate the density of the mixture using the partial molar volumes and
-     * mole fractions as input
-     *
-     * The formula for this is
-     *
-     * @f[
-     * \rho = \frac{\sum_k{X_k W_k}}{\sum_k{X_k V_k}}
-     * @f]
-     *
-     * where @f$ X_k @f$ are the mole fractions, @f$ W_k @f$ are the molecular
-     * weights, and @f$ V_k @f$ are the pure species molar volumes.
-     *
-     * Note, the basis behind this formula is that in an ideal solution the
-     * partial molar volumes are equal to the pure species molar volumes. We
-     * have additionally specified in this class that the pure species molar
-     * volumes are independent of temperature and pressure.
-     *
-     * NOTE: This is a non-virtual function, which is not a member of the
-     *       ThermoPhase base class.
-     */
```

Comment:
Comment may be worth keeping?

Reply:
This is identical to the documentation of the method that is being overridden ([VPStandardStateTP::calcDensity](https://github.com/speth/cantera/blob/e7862d20b4f03eff084c0491725240e2ea043045/include/cantera/thermo/VPStandardStateTP.h#L165-L185))